### DISCUSSION TITLE
Tailwind and Angular 15. Form style dont work after reload page.

### DISCUSSION THREAD

**Discussion by CristianVallecchia** [tailwindlabs/tailwindcss#11061] [Help]
In my application, when I first navigate to a form, the style doesn't load. To make it work I need to do a refresh.

I use angular 15, the latest version of Tailwind, node 18.13.0
scss.

My config.

`/** @type {import('tailwindcss').Config} */
module.exports = {
  content: ['./src/**/*.{html,ts}'],
  theme: {
    extend: {
      colors: {
        primary: '#0B98AC',
        accent: '#7600AD',
        secondary: '#FF992E',
        'action-green': '#519C66',
        'action-red': '#A42323'
      },
      screens: {
        '3xl': '1600px'
      }
    }
  },
  plugins: []
};`

**Comment by wongjn** [comment]
It seems like this would be some misconfiguration with Angular. Try requesting assistance in a discussion area targeted towards Angular – you may be able to get more targeted help there.
